flux.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: flux
+  version: 0.38.2
+  epoch: 0
+  description: Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - build-base
+      - bash
+      - curl
+      - go
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/fluxcd/flux2/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 6f64ac68748137816e85b2258681782951b04135d00d8e69e0efa96438f47177
+  - runs: |
+      curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+      echo "The current working directory is $(pwd)"
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      install -Dm755 $(pwd)/kustomize /usr/bin/kustomize
+      echo "The kustomize version is $(kustomize version)"
+      make build
```

Comment:
We need to set the `VERSION` arg for `make build` to produce a valid binary, otherwise people can’t use it to bootstrap Flux.